components/colorful-component/colorful-component.tsx

```diff
@@ -0,0 +1,38 @@
+import cn from "classnames";
+import styles from "./colorful-component.module.scss";
+import { IColorfullComponent } from "../../fixtures/colorful-component.fixtures";
+
+const ColorfulComponent = ({ textTitle, bgTitle, bgColor }: IColorfullComponent) => {
+
+  const getBgColors = () => {
```

Comment:
This should be simplified.

Instead of trying to add n boolean conditions, grab the CSS style rule by matching `styles[color]` where `color` is passed in from what props got